### PR TITLE
PSMDB-1187 Release notes 4.4.18-18

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: docs/release_notes/4.4.18-18.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* [SERVER-66289](https://jira.mongodb.org/browse/SERVER-66289) - Fixed the issue with how the server handles batches of writes when running $out with secondary read preference by updating write size estimation logic in ``DocumentSourceWriter``
+* [SERVER-61185](https://jira.mongodb.org/browse/SERVER-61185) - Improved the performance of inserts into unique indexes
+* [SERVER-68115](https://jira.mongodb.org/browse/SERVER-68115) - Prevented dropping empty path component from elemMatch path during index selection
+* [SERVER-50454](https://jira.mongodb.org/browse/SERVER-50454) - Avoided sending the "keyValue" field to drivers on duplicate key error
+* [SERVER-69443](https://jira.mongodb.org/browse/SERVER-69443) - Disallowed the use of the ``allowSpeculativeMajorityReads`` flag for the ``find`` command in transactions
+
```
Comment (igorsol) [inline docs/release_notes/4.4.18-18.md:22]: Please add improvement PSMDB-1181

Comment (nastena1606) [reply docs/release_notes/4.4.18-18.md:22]: Added

**Review by sandraromanchenko — APPROVED** [review]

**Review by igorsol — APPROVED** [review]
LGTM

**Review by janwieremjewicz — APPROVED** [review]
Looks great!

**Pull request merged**